bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -0,0 +1,2172 @@
+				{ name: "type1", type: "STRING" },
+				{ name: "type2", type: "STRING" },
+				{ name: "type3", type: "STRING" },
+				{ name: "type4", type: "STRING" },
+				{ name: "type5", type: "STRING" },
+				{ name: "type6", type: "STRING" },
+				{ name: "refreshEndTime", type: "STRING" },
+				{ name: "endTime", type: "STRING" },
+				{ name: "userId", type: "STRING", required: true },
+				{ name: "endDate", type: "STRING" },
+				{ name: "dayOfMonth", type: "STRING" },
+				{ name: "days", type: "STRING" },
+				{ name: "location", type: "STRING" },
+				{ name: "refreshStartTime", type: "STRING" },
+				{ name: "recurrenceType", type: "STRING" },
+				{ name: "reccurEndDate", type: "STRING" }
+			],
+			function: editRecurringEventwithCustomTimezone
+		},
+
+		{
+			name: "EditRecurringOfficeHoursWithCurrentTimezone",
+			parameters: [
+				{ name: "isOnlineMeeting", type: "BOOLEAN" },
+				{ name: "subject", type: "STRING" },
+				{ name: "timezone", type: "STRING" },
+				{ name: "startDate", type: "STRING" },
+				{ name: "startDateTime", type: "STRING" },
+				{ name: "content", type: "STRING" },
+				{ name: "id", type: "STRING", required: true },
+				{ name: "refreshEndTime", type: "STRING" },
+				{ name: "endDateTime", type: "STRING" },
+				{ name: "userId", type: "STRING", required: true },
+				{ name: "endDate", type: "STRING" },
+				{ name: "dayOfMonth", type: "STRING" },
+				{ name: "days", type: "STRING" },
+				{ name: "refreshStartTime", type: "STRING" },
+				{ name: "recurrenceType", type: "STRING" },
+			],
+			function: editRecurringOfficeHoursWithCurrentTimezone
+		},
+
+		{
+			name: "EditRecurringOfficeHoursWithCustomTimezone",
+			parameters: [
+				{ name: "isOnlineMeeting", type: "BOOLEAN" },
+				{ name: "subject", type: "STRING" },
+				{ name: "timezone", type: "STRING" },
+				{ name: "startDate", type: "STRING" },
+				{ name: "startTime", type: "STRING" },
+				{ name: "content", type: "STRING" },
+				{ name: "id", type: "STRING", required: true },
+				{ name: "refreshEndTime", type: "STRING" },
+				{ name: "endTime", type: "STRING" },
+				{ name: "userId", type: "STRING", required: true },
+				{ name: "endDate", type: "STRING" },
+				{ name: "dayOfMonth", type: "STRING" },
+				{ name: "days", type: "STRING" },
+				{ name: "refreshStartTime", type: "STRING" },
+				{ name: "recurrenceType", type: "STRING" },
+				{ name: "reccurEndDate", type: "STRING" }
+			],
+			function: editRecurringOfficeHoursWithCustomTimezone
+		},
+
+		{
+			name: "refresh",
+			parameters: [
+				{ name: "endDateTime", type: "STRING" },
+				{ name: "startDateTime", type: "STRING" },
+				{ name: "id", type: "STRING", required: true }
+			],
+			function: refresh
+		}
+
+	]
+});
+
+
+const moment = library.load('moment-timezone');
+const uuid = library.load('uuid');
+const pageSize = 100;
+const startDate = moment.utc().subtract(1, 'd').format();
+
+async function fullSync({ client, dataStore }) {
+	let userId = [];
```

Comment:
This array will contain only one record at a time ?, if not rename it as `userIds` for better code readability. 
```suggestion
	const userIds = [];
```

Reply:
Sure, Will update the code.